UPGRADE.md

```diff
@@ -8,6 +8,12 @@ Override `src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twi
     - Interface `Sylius\InvoicingPlugin\Provider\ChannelColorProviderInterface` was removed.
     - Class `Sylius\InvoicingPlugin\Twig\ChannelColorExtension` and twig filter `sylius_channel_color` were removed.
     - Parameter `default_channel_color` was removed
+4. Way to configure resources has been changed:
+    ```dif
```

Comment:
```suggestion
    ```diff
```

Reply:
And this is not an only change, there is also a need to change the resource name, as for example for invoice:
```diff
-invoice:
+sylius_invoicing_plugin.invoice:
```